Packs/IronDefense/Integrations/IronDefense/IronDefense.py

```diff
@@ -301,14 +689,272 @@ def report_observed_bad_activity_command():
     demisto.results(results)
 
 
+def get_event_command():
+    # get event data from IronAPI
+    event_id = demisto.getArg('event_id')
+    results = IRON_DEFENSE.get_event(event_id)
+
+    # Output the event data
+    event = results.get('event')
+    vue_markdown_link = IRON_DEFENSE.create_markdown_link("Open in IronVue", event.get("vue_url"))
```

Comment:
why do we need the link to the original UI? can't we get the results here?

Reply:
While we try to provide full feature parody with our integrations we've maintained a link back to our UI as an option.